core/templates/dev/head/pages/email_dashboard/email_dashboard_result.html

```diff
@@ -46,6 +46,7 @@
   </style>
   <div class="oppia-email-form" ng-controller="EmailDashboardResult">
     <p ng-if="emailSubmitted" class="success">Emails sent successfully. Redirecting to the Email Dashboard...</p>
+    <p ng-if="testEmailSentSuccessfully" class="success">Test email is sent to your email successfully.</p>
```

Comment:
User-readable text should be something like: "Test email sent successfully. Please check your inbox."